src/python/pants/backend/javascript/target_types.py

```diff
@@ -44,6 +45,9 @@ class JSSourceTarget(Target):
 
 class JSSourcesGeneratorSourcesField(JSGeneratorSourcesField):
     default = tuple(f"*{ext}" for ext in JS_FILE_EXTENSIONS)
+    help = generate_multiple_sources_field_help_message(
```

Comment:
🤯 
I always thought the help auto-populated the defaults. I recall asking about it, and I guess I forgot the correct answer 🤦🏽‍♂️

Reply:
`help` used to be provided for `MultipleSourcesField`, but https://github.com/pantsbuild/pants/pull/15633 changed it so now you have to set it so that we can have better examples in the message. I think we forgot to update your code!